### DISCUSSION TITLE
Feature request: User profile status editing

### DISCUSSION THREAD

**Discussion by GuillemRoca** [community/community#113602] [Mobile] [answered]
Hello!

I've been using the app since it was released and one thing I found missing is to edit the user profile status from the mobile app.

I find this feature especially useful if paired to the busy state setting which can help you indicate to your team you are on vacation/busy to avoid getting assigned for PRs.

Since this feature was [introduced 5 years ago](https://github.blog/changelog/2019-01-09-set-your-status/) on the web I thought it could be nice to add it to the app for ease of use.

I can always log in using the web to change this setting but since with the app, I'm already authenticated that would streamline the experience.

Keep up the amazing work! 🚀

**Comment by kangarruu** [comment]
Hello, thanks for writing in and for being a part of the GitHub Mobile community. We've updated the tracking issue for this to help get it on our roadmap. I agree that this feature would be very handy!

**Reply by GuillemRoca** [reply]
Amazing! Thanks for the quick reply.

By your response I'm guessing that there is still no task in the roadmap to follow it's progress. Could you add it here for reference whenever it is so we could potentially close this discussion?

Thanks! :)

**Reply by kangarruu** [reply]
There was an internal tracking issue for this 🙂. Stay tuned! 😉

**Reply by shfhg587-lang** [reply]
noreply@github.com

**Comment by GuillemRoca** [comment, marked as answer]
I can't recall when the functionality was added but I can now set the busy state in the profile using GitHub Mobile app! 🥳

Thanks for implementing the feature, works great!

**Comment by j4sg8hmgw9-design** [comment]
Hello
